src/kfactory/kcell.py

```diff
@@ -428,10 +428,7 @@ def __init__(
         self._settings: KCellSettings = KCellSettings()
         self.info: Info = Info()
         self._locked = False
-        if name is None:
-            _name = "Unnamed_!"
-        else:
-            _name = name
+        _name = "Unnamed_!" if name is None else name
```

Comment:
Please don't touch. This is a very sensitive piece of code that I debugged more times than I want to admit

Reply:
sure :)